### ISSUE TITLE
Some characters are not encoded correctly

### ISSUE THREAD

**Issue by TheOnlyZac** [TheOnlyZac/sly-string-toolkit#1]
Some characters, such as ®️, are being encoded incorrectly and are showing up with extra extraneous characters in game.

**Comment by TheOnlyZac** [comment]
Switching the encoding from UTF-8 to Ascii with no other changes yields the following error:
```
Traceback (most recent call last):
  File "main.py", line 91, in <module>
    main()
  File "main.py", line 24, in main
    string_pnach, string_pointers = strings.generate_strings_pnach(args.input, int(args.address, 16))
  File "F:\Games\Sly\string-replacement\strings.py", line 25, in generate_strings_pnach
    strings.append((int(row[0]), row[1].encode('ascii') + b'\x00'))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 36-37: ordinal not in range(128)
```

**Comment by TheOnlyZac** [comment]
Fixed in #7
